.ci/jsoref-spellchecker/whitelist.words

```diff
@@ -1184,8 +1189,11 @@ SETLABELFOR
 SETPOINT
 sevntu
 SGSU
+shellcheck
+shellcheckrc
 shengchen
 Shiell
+shopt
```

Comment:
please remove from this file all non-required changes

Reply:
Changes Done